src/Compilers/Core/CodeAnalysisTest/Collections/ImmutableSegmentedListBuilderTest.cs

```diff
@@ -380,7 +380,7 @@ public void ItemRef()
             var builder = new ImmutableSegmentedList<int>.Builder(list);
 
             ref readonly var safeRef = ref builder.ItemRef(1);
-            ref var unsafeRef = ref Unsafe.AsRef(safeRef);
+            ref var unsafeRef = ref Unsafe.AsRef(in safeRef);
```

Comment:
Backport diagnostic fix #70248

Reply:
This shouldn't be needed. It's caused by an allowed roll-forward in global.json; setting it to disable removes the need to make this change.